packages/frontend/src/view/ApiDefs/Form/CacheTransforms.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { CacheTransforms as Props } from '../../../types';
+import { Col, FormGroup, Input, OutlineButton, Row } from '../../../ui';
+import { AddFieldArrayHeader } from '../../Form/AddFieldArrayHeader';
+import { useStyles } from './useStyles';
+import { CacheTransformsInvalidateOperations } from './CacheTransformsInvalidateOperations';
+
+export const CacheTransforms: React.FC<Props> = ({
+  cache,
+  onAddCache,
+  onRemoveCache,
+  register,
+  errors,
+  control,
+}) => {
+  const { formRow } = useStyles();
+
+  return (
+    <FormGroup title="Cache Transforms">
+      <AddFieldArrayHeader title="Cache Transform" onAddClick={onAddCache} />
+      {cache.map((cache: any, idx: number) => {
+        return (
+          <FormGroup key={`cacheTransform-${idx}`} title="Cache Transform">
+            <Row spacing={2} className={formRow}>
+              <Col xs={6}>
+                <Input
+                  ref={register()}
+                  name={`cache[${idx}].field`}
+                  label="Field"
+                  error={!!errors?.cache?.[idx]?.field}
+                  defaultValue={cache.field || undefined}
+                  fullWidth
+                  required
+                />
+              </Col>
+            </Row>
+            <Row spacing={2} className={formRow}>
+              <Col xs={6}>
+                <Input
+                  ref={register()}
+                  name={`cache[${idx}].cacheKey`}
+                  label="Custom cache key"
```

Comment:
helperText="Cache key used to store the responses. Default is {typeName}-{fieldName}-{argsHash}-{fieldNamesHash}"

Reply:
@Feshchenko it would be great to add a "?" icon at the end of the helper text, which will show a tooltip on hover with the following content:
```
Available variables:

{args.argName} - use resolver argument
{typeName} - use name of the type
{fieldName} - use name of the field
{argsHash} - a hash based on the 'args' object
{fieldNamesHash} - a hash based on the field names selected by the client
{info} - the GraphQLResolveInfo of the resolver
```